packages/fxa-auth-server/lib/routes/password.js

```diff
@@ -56,12 +57,14 @@ module.exports = function (
         log.begin('Password.changeStart', request);
         const form = request.payload;
         const oldAuthPW = form.oldAuthPW;
+        let uid='';
```

Comment:
Linting error:
```sh
➤ YN0000: [fxa-auth-server]: /home/circleci/project/********/fxa-auth-server/lib/routes/password.js
➤ YN0000: [fxa-auth-server]:   60:13  error  'uid' is assigned a value but never used  @typescript-eslint/no-unused-vars
```

Reply:
Also a bit strange that there isn't a space around the `=`. Wondering if Prettier hooks aren't prettier-ing these files. (but that's an almost unmentionably tiny uber nit, although does potentially mean we might get reformatting of the code the next time somebody edits+saves this file)